student.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
```

Comment:
### Optional
- [ ] It would be nice to use `require_relative` for files in our project directory